ports/libxpm/portfile.cmake

```
@@ -0,0 +1,19 @@
+vcpkg_from_github(
```

```suggestion
vcpkg_check_linkage(ONLY_STATIC_LIBRARY)

vcpkg_from_github(
```

You can prevent dll generation using `vcpkg_check_linkage`. This allows to build the port with triplets that build both static and dynamic libraries.